### DISCUSSION TITLE
Tabular Data Display in HyperSwitch Portal Needs to be Revisited

### DISCUSSION THREAD

**Discussion by nara-golfdistrict-com** [juspay/hyperswitch#3971] [Ideas / Feature Requests]
The tabular data in the HyperSwitch portal is very hard to work with and below are some of the challenges.
1. The table is not wide enough to leverage the full width of the screen. I have logged a separate feature request for this.
2. There is no sorting in the header. I have logged a separate feature request for this.
3. Filtering does not filter by all column values. I have logged a separate feature request for this.
4. There is no horizontal scrollbar for us to see all the columns,
5. The column order is fixed and not rearrangeable.
6. The date times are in AKDT. It should all be in local timezone. If you can't automatically detect the local timezone, then please provide an option to select the timezone in which we want to see the transactions. 
7. Have frozen columns.
8. No way to reset the column selection to where it was before.
9. There should be an easy way to download the data from the UI. Currently an email is being sent.
10. One should be able to see more than 10 records in a page. The number of records displayed should be a selection in the page.
See below screenshots.
![image](https://github.com/juspay/hyperswitch/assets/145275696/90a1608f-07d2-4545-805e-f888102a883b)
<img width="2073" alt="image" src="https://github.com/juspay/hyperswitch/assets/145275696/c8d0b1ba-1a37-4508-bf41-4911f964945e">

**Comment by dvenga** [comment]
Thanks @nara-golfdistrict-com - for the comprehensive feedback.

We had identified some of these issues but other feature requests was prioritised over this. We will start picking some of the items from this list in this sprint and close - so that the experience becomes easier. 

We will keep updating this discussion with the updates.
